docs/hub/spaces-sdks-docker-chatui.md

```diff
@@ -0,0 +1,33 @@
+# Chat UI on Spaces
+
+**Hugging Chat** is the interface of Hugging Face enabling everyone to try open-source large language models using Hugging Face's infrastructure. This Docker template enables everyone to deploy their own Hugging Chat based on a model of their choice with a few clicks.
```

Comment:
WDYT of these additions?
```suggestion
**Hugging Chat** is an open-source interface enabling everyone to try open-source large language models such as Falcon, StarCoder and BLOOM. Thanks to an official Docker template, you can deploy your own Hugging Chat based on a model of your choice with a few clicks using Hugging Face's infrastructure.
```

Reply:
or better the other way around:

> chat-ui is an open source interface [...] It is the codebase that powers HuggingChat: (link to huggingchat)

Also please mention OpenAssistant (and LLaMA too)